src/librustc/mir/interpret/error.rs

```diff
@@ -20,7 +20,7 @@ pub struct EvalError<'tcx> {
 
 impl<'tcx> From<EvalErrorKind<'tcx>> for EvalError<'tcx> {
     fn from(kind: EvalErrorKind<'tcx>) -> Self {
-        let backtrace = match env::var("RUST_BACKTRACE") {
+        let backtrace = match env::var("MIRI_BACKTRACE") {
             Ok(ref val) if !val.is_empty() => Some(Backtrace::new_unresolved()),
```

Comment:
So, it looks like backtrace will be generated even with `RUST_BACKTRACE=0`. Is this okay?

Reply:
Yes. This is not your regular backtrace, this is inside miri for debugging miri error sources